src/data/ebkpData.ts

```
@@ -5,46 +5,78 @@ export interface EbkpDataItem {
+  if (match) {
+    const [, letter, group, element] = match;
+    // Pad with leading zeros to ensure 2 digits
+    const paddedGroup = group.padStart(2, '0');
+    const paddedElement = element.padStart(2, '0');
+    return `${letter}${paddedGroup}.${paddedElement}`;
+  }
+  return code; // Return original if it doesn't match the pattern
+};
+
+// Prebuilt Map for faster lookups
+const ebkpMap = new Map(ebkpData.map(i => [i.code, i.name]));
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)